include/rssitem.h

```diff
@@ -183,14 +189,16 @@ class RssItem : public Matchable {
 	std::string feedurl_;
 	Cache* ch;
 	std::string enclosure_url_;
-	std::string enclosure_type_;
+	std::string enclosure_mime_type_;
 	std::string flags_;
 	std::string oldflags_;
 	std::weak_ptr<RssFeed> feedptr_;
 	std::string base;
 	unsigned int idx;
 	unsigned int size_;
 	time_t pubDate_;
+	// There is no setter for this; it's set by `set_enclosure_mime_type()`
+	LinkType enclosure_link_type_;
```

Comment:
I think it is better to call `utils::podcast_mime_to_link_type()` in `enclosure_link_type()`.
That makes sure it cannot get out of sync with the configured mimetype and reduces the need for a comment.

Reply:
I think this comment is outdated now that I pushed https://github.com/newsboat/newsboat/pull/854/commits/8ea0b9802aadd4c91f240990867f1975be1864a6.